### PR TITLE
🦕 Edit translations page

### PR DESCRIPTION
**Pull request by lucianatl**
## Description

added a grid for the text to be translated, a textbox for the translation and a checkbox for verification
progress bar and submit bar at the bottom in accordance w figma

Fixes #<number>

## Screenshots
image

### CODE REVIEW

**Review by jackieo5023 — COMMENTED** [review]
image
It looks like there's some wonky stuff going on with spacing on different window sizes - for now we can just ignore the checkbox, but for the english text it looks like it isn't in line with the "English" title. Maybe we can also add some padding in between the english text and the divider? Also if you could figure out what's going on with the footer too that would be good 😓 

In general, I think it would be good to go by the rule of: If you add margin/padding/any type of spacing to the "title" grid, do the same to the actual "translation" grid (and vice versa).

**Comment by jackieo5023** [comment]
One thing I noticed when messing around with the CSS - the `justify-center` attribute messes up the "English" column when it's there, but when you get rid of it, the text box for "Spanish" is misaligned - if you add the `margin-left: 10%` to the "Spanish" text box, it would be fixed (at least from what I tested).

File: client/src/css/EditTranslations.css [hunk 1]
```diff
@@ -0,0 +1,86 @@
+.edit-tranlsations .header {
```
Comment (Lam7150) [inline client/src/css/EditTranslations.css:1]: translations*

File: client/src/pages/EditTranslations.jsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
+// @flow
```
Comment (Lam7150) [inline client/src/pages/EditTranslations.jsx:1]: delete this comment

**Review by Lam7150 — APPROVED** [review]
Looks good!

**Pull request merged**